### DISCUSSION TITLE
Welcome to ECUBus Discussions!

### DISCUSSION THREAD

**Discussion by frankie-zeng** [ecubus/EcuBus-Pro#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JOSEPH129009** [comment]
Hi Zeng, May I ask how do I inplement crc check funtion in the ECUBus

**Reply by frankie-zeng** [reply]
there is not a build in CRC method in ECUBus, but you can use implement it in the script. use a CRC table, copy [example code](https://github.com/alexgorbatchev/cr) into script.

**Reply by JOSEPH129009** [reply]
Thank you! I found some examples from NXP as well. It is a really awesome UI for uds. By the way, is it possible to receive 8 byte seed from the security request seed. I tried it. It receives no more than one frame
